<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="dateTimeLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="convertPushButton">
            <property name="text">
             <string>Umwandeln</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="addSubComboBox">
          <item>
           <property name="text">
            <string>Hinzuf√ºgen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Entfernen</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="kindComboBox">
          <item>
           <property name="text">
            <string>Stunden</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Minuten</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sekunden</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tage</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Monate</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jahre</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="addSubPushButton">
          <property name="text">
           <string>berechnen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="resultLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QDateEdit" name="fromDateEdit">
          <property name="displayFormat">
           <string>dd.MM.yyyy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="toDateEdit">
          <property name="displayFormat">
           <string>dd.MM.yyyy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>berechnen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="daysLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>QTime:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTimeEdit" name="timeEdit">
            <property name="displayFormat">
             <string>HH:mm:ss</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>QDate:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateEdit" name="dateEdit">
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>QDateTime</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDateTimeEdit" name="dateTimeEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Kalender:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCalendarWidget" name="calendarWidget"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>timeEdit</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>dateTimeEdit</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>dateTimeEdit</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateTimeEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>dateEdit</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateTimeEdit</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>timeEdit</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateTimeEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>calendarWidget</receiver>
   <slot>setSelectedDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>calendarWidget</receiver>
   <slot>setSelectedDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calendarWidget</sender>
   <signal>clicked(QDate)</signal>
   <receiver>dateTimeEdit</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calendarWidget</sender>
   <signal>clicked(QDate)</signal>
   <receiver>dateEdit</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
